src/AccessibilityInsights.Extensions.AzureDevOps/AzureDevOpsIntegration.cs

```diff
@@ -491,27 +491,43 @@ internal Uri CreateIssuePreview(string projectName, string teamName, IReadOnlyDi
         }
 
         /// <summary>
-        /// Retains characters within ascii range below 127 and returns
-        /// an escaped representation of the string
+        /// Substitutions for characters when creating the bug URL
+        /// </summary>
+        internal static IReadOnlyDictionary<char, char> CharacterSubstitutions = new Dictionary<char, char>()
+        {
+            { '�', '-' }, // middle dot (183) to dash (45)
+        };
+
+        /// <summary>
+        /// replaces characters if substitution defined and
+        /// returns an escaped representation of the string
         /// </summary>
         /// <param name="str"></param>
         /// <returns></returns>
         internal static string EscapeForUrl(string str)
         {
+            // characters such as the middle dot (ascii 183) can 
+            // cause issues during navigation, so we only allow
+            // a basic set of characters when building the URL
             var sb = new StringBuilder();
             foreach (var c in str)
             {
-                // characters such as the middle dot (ascii 183) can 
-                // cause issues during navigation, so we only include
-                // a basic set of characters when building the URL
-                if (c <= 127)
+                if (CharacterSubstitutions.TryGetValue(c, out char replaced))
```

Comment:
>replaced [](start = 67, length = 8)

I'd probably call this substitute, but it's not a big deal

Reply:
If it's similar to you either way, I prefer the noun form; plan to leave as-is

Reply:
"replaced" isn't a noun. "replacement" or "substitution" are nouns (and "substitution" has the benefit of matching the collection name)

---
In reply to: [277098982](https://github.com/Microsoft/accessibility-insights-windows/pull/323#discussion_r277098982) [](ancestors = 277098982)

Reply:
substitute is commonly used as a noun, as well